src/main/java/com/epam/tc/hw7/components/MetalsColorsForm.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw7.components;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.tc.hw7.entities.MetalsColors;
+
+public class MetalsColorsForm extends Form<MetalsColors> {
+
+    @UI("section#odds-selector input") RadioButtons oddsSelector;
```

Comment:
usually filed is located on the next line for readability

Reply:
I agree with you - it really will be more readable